package/kedro_viz/server.py

```diff
@@ -228,16 +243,43 @@ def _call_viz(host=None, port=None, browser=None, load_file=None, save_file=None
                 click.echo("Invalid file, top level key '{}' not found.".format(key))
```

Comment:
I believe it's better to raise `kedro.cli.utils.KedroCliError` (or `ClickException` if you don't want to rely on kedro exception) rather than exiting yourself.